pkg/assetfetcher/http.go

```diff
@@ -26,7 +26,7 @@ const (
 // SiteAssetsResponse is the structure of the Nexpose site assets response
 type SiteAssetsResponse struct {
 	// Hypermedia links to corresponding or related resources
-	Links Link
+	Links []Link
```

Comment:
Question: do we use the Links at all? Could we just drop this from the struct instead?

Reply:
That is a _great_ question! We might just be able to drop this from the struct indeed, let me double check.

Reply:
That is a _great_ question! We might just be able to drop this from the struct indeed, let me double check.

Reply:
Yep, Links can be removed, I'll remove it, thanks @bkrebsbach